README.rst

```diff
@@ -28,7 +28,7 @@ The are accessible through separate URL (you can click `View on the site` link
 while editing).
 
 
-Page generating
+Page generator
```

Comment:
I think this is probably even better modified to something like "Generating pages" or "Page generation" since it's talking about the action of generating a page. What do you think?